### PR TITLE
Add the description of sysvar `max_allowed_packet`

### PR DESCRIPTION
**Pull request by CbcWestwolf**
### What is changed, added or deleted? (Required)

Add the description of sysvar `max_allowed_packet`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -268,6 +268,13 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 1 row in set (0.00 sec)
 ```
 
+### `max_allowed_packet`
+
+* 作用域：SESSION | GLOBAL
+* 默认值：67108864
+* 范围：`[1024, 1073741824]`，且必须为 1024 的整数倍
+* 服务器端和客户端在一次传送数据包的过程中允许最大的数据包大小，单位为字节
```
Comment (CbcWestwolf) [inline system-variables.md:276]:
```suggestion
* 服务器端和客户端在一次传送数据包的过程中允许最大的数据包大小，单位为字节。
```

File: system-variables.md [hunk 2]
```diff
@@ -268,6 +268,13 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 1 row in set (0.00 sec)
 ```
 
+### `max_allowed_packet`
+
+* 作用域：SESSION | GLOBAL
+* 默认值：67108864
+* 范围：`[1024, 1073741824]`，且必须为 1024 的整数倍
+* 服务器端和客户端在一次传送数据包的过程中允许最大的数据包大小，单位为字节。
```
Comment (bb7133) [inline system-variables.md:276]: 可以说明一下，这个变量与 MySQL 中语义一致？

Comment (CbcWestwolf) [reply system-variables.md]: Done :-)

File: system-variables.md [hunk 3]
```diff
@@ -268,6 +268,14 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 1 row in set (0.00 sec)
 ```
 
+### `max_allowed_packet`
```
Comment (shichun-0415) [inline system-variables.md:271]: 请提供该变量引入版本

Comment (CbcWestwolf) [reply system-variables.md]: Done :-)

File: system-variables.md [hunk 4]
```diff
@@ -268,6 +268,14 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 1 row in set (0.00 sec)
 ```
 
+### `max_allowed_packet` <span class="version-mark">从 v6.1.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 默认值：67108864
+- 范围：`[1024, 1073741824]`，且必须为 1024 的整数倍
```
Comment (TomShawn) [inline system-variables.md:275]:
```suggestion
- 取值范围：`[1024, 1073741824]`，且必须为 1024 的整数倍
```

File: system-variables.md [hunk 5]
```diff
@@ -268,6 +268,14 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
+- 服务器端和客户端在一次传送数据包的过程中允许最大的数据包大小，单位为字节。
```
Comment (TomShawn) [inline system-variables.md:276]:
```suggestion
- 服务器端和客户端在一次传送数据包的过程中所允许最大的数据包大小，单位为字节。
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: system-variables.md [hunk 6]
```diff
@@ -268,6 +268,14 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 1 row in set (0.00 sec)
 ```
 
+### `max_allowed_packet` <span class="version-mark">从 v6.1.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 默认值：67108864
```
Comment (TomShawn) [inline system-variables.md:274]:
```suggestion
- 默认值：67108864
- 是否持久化到集群：是
```

File: system-variables.md [hunk 7]
```diff
@@ -268,6 +268,15 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 1 row in set (0.00 sec)
 ```
 
+### `max_allowed_packet` <span class="version-mark">从 v6.1.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：67108864
+- 取值范围：`[1024, 1073741824]`，且必须为 1024 的整数倍
```
Comment (bb7133) [inline system-variables.md:276]:
```suggestion
- 取值范围：`[1024, 1073741824]`，且应当为 1024 的整数倍；若无法被 1024 整除，则会提示 warning 并向下取整：例如设置为 1025，则最终结果为 1024。
```

Comment (TomShawn) [reply system-variables.md:276]:
```suggestion
- 取值范围：`[1024, 1073741824]`，且应当为 1024 的整数倍；若取值无法被 1024 整除，则会提示 warning 并向下取整。例如设置为 1025 时，则 TiDB 中的实际取值为 1024。
```

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**